FluentAssertions.Core/Equivalency/EquivalencyAssertionOptions.cs

```diff
@@ -101,7 +101,7 @@ public class EquivalencyAssertionOptions : EquivalencyAssertionOptionsBase<Equiv
         public EquivalencyAssertionOptions()
         {
             IncludingNestedObjects();
-            IncludingAllDeclaredProperties();
+            IncludingAllDeclaredMembers();
```

Comment:
I should note that it follows the same "NonPublic" behavior used with properties.  It my knowledge `ShouldBeEquivalentTo` does not have any built in support for private members.  What I don't know off the top of my head is how you classified `internal` and `protected` with respect to non-public.  I can check quick.